library/init/logic.lean

```diff
@@ -521,8 +521,15 @@ iff.intro (λ h, trivial) (λ ha h, false.elim h)
 @[simp] theorem true_implies_iff (α : Prop) : (true → α) ↔ α :=
 iff.intro (λ h, h trivial) (λ h h', h)
 
-/- exists -/
+/--
+The existential quantifier.
 
+To prove a goal of the form `⊢ ∃ x, p x`, you can provide a witness `x` with the tactic `use x`.
+You'll then be left with the goal `⊢ p x`.
```

Comment:
This could perhaps be made even more clear by not using the variable `x` into two different ways:
```suggestion
To prove a goal of the form `⊢ ∃ x, p x`, you can provide a witness `y` with the tactic `use y`.
You'll then be left with the goal `⊢ p y`.
```

Reply:
I'm confused. Isn't your `y` exactly the `x` in the goal?

Reply:
The idea is to differentiate between the variable bound by the existential quantor and the witness.

Reply:
I've had professional mathematicians be confused about this during demo talks.